decidim-debates/spec/system/admin_manages_debates_spec.rb

```diff
@@ -15,4 +20,5 @@
   it_behaves_like "manage debates"
   it_behaves_like "manage announcements"
```

Comment:
It seems with this one we have the announcements specs failing:
```
$ cd decidim-debates
$ bundle exec rspec spec/system/admin_manages_debates_spec.rb
```

Not quite sure what is causing that but it does seem to be consistently failing (re-ran it twice, same error).

Reply:
This is solved with the latest merge 😄 
Can we merge this?